### PR TITLE
update initCCP documentation

### PR DESCRIPTION
**Pull request by seiyako**

### CODE REVIEW

File: Documentation.md [hunk 1]
```diff
@@ -154,15 +154,15 @@ everything set up correctly and that you are able to listen for events.
           },
           region: "eu-central-1",         // REQUIRED for `CHAT`, optional otherwise
           softphone: {                    // optional
```
Comment (ctwomblyamzn) [inline Documentation.md:156]: maybe the comment on this should be "optional, default below apply if not provided"

Comment (seiyako) [reply Documentation.md:156]: makes sense, will add it

File: Documentation.md [hunk 2]
```diff
@@ -153,16 +153,16 @@ everything set up correctly and that you are able to listen for events.
...
-            allowFramedSoftphone: true,   // optional
-            disableRingtone: false,       // optional
-            ringtoneUrl: "./ringtone.mp3" // optional
-           },
-           pageOptions: { //optional
-            enableAudioDeviceSettings: false, //optional, defaults to 'false'
-            enablePhoneTypeSettings: true //optional, defaults to 'true' 
-           }       
-         });
+          softphone: {                    // optional, defaults below apply if not provided"
```
Comment (ctwomblyamzn) [inline Documentation.md:156]: tiny nit: unnecessary quote

**Review by ctwomblyamzn — COMMENTED** [review]
Can we also include these changes in README please?

File: Documentation.md (hunk 2 above)
Comment (seiyako) [reply Documentation.md:156]: oh

**Comment by ctwomblyamzn** [comment]
> Can we also include these changes in README please?

Ignore me...

File: README.md [hunk 3]
```diff
@@ -155,16 +155,16 @@ everything setup correctly and that you will be able to listen for events.
...
-            allowFramedSoftphone: true,   // optional
-            disableRingtone: false,       // optional
-            ringtoneUrl: "./ringtone.mp3" // optional
-           },
-          pageOptions: { //optional
-            enableAudioDeviceSettings: false, //optional, defaults to 'false'
-            enablePhoneTypeSettings: true //optional, defaults to 'true' 
-           }  
-         });
+          softphone: {                    // optional, defaults below apply if not provided"
```
Comment (ctwomblyamzn) [inline README.md:158]: still there in README

**Review by ctwomblyamzn — APPROVED** [review]

**Pull request merged**